src/test/unit/views/mutiple-respondent-check.test.ts

```diff
@@ -0,0 +1,105 @@
+import { expect } from 'chai';
+import { app } from '../../../main/app';
+import request from 'supertest';
+
+import fs from 'fs';
+import path from 'path';
+
+const multipleRespondentJsonRaw = fs.readFileSync(path.resolve(__dirname, '../../../main/resources/locales/en/translation/multiple-respondent-check.json'), 'utf-8');
+const commonJsonRaw = fs.readFileSync(path.resolve(__dirname, '../../../main/resources/locales/en/translation/common.json'), 'utf-8');
+const multipleRespondentJson = JSON.parse(multipleRespondentJsonRaw);
+const commonJsonRawJson = JSON.parse(commonJsonRaw);
+
+const PAGE_URL = '/multiple-respondent-check';
+const titleClass = 'govuk-heading-xl';
+const pClass = 'govuk-body';
+const iClass = 'govuk-inset-text';
+const expectedTitle = multipleRespondentJson.h1;
+const expectedP1 = multipleRespondentJson.p1;
+const expectedP2 = multipleRespondentJson.p2;
+
+const buttonClass = 'govuk-button';
+const backButtonClass = 'govuk-back-link';
+const radioClass = 'govuk-radios__item';
+const errorClass = 'govuk-error-message';
+const errorSummaryClass = 'govuk-error-summary__list';
+const errorSummaryTitleClass = 'govuk-error-summary__title';
+const expectedRadioLabel1 = multipleRespondentJson.radio1;
+const expectedRadioLabel2 = multipleRespondentJson.radio2;
+const expectedButtonText = commonJsonRawJson.continue;
+const expectedBackButtonText = commonJsonRawJson.back;
+const expectedErrorMessage = multipleRespondentJson.errorMessage;
+const errorSummaryTitle = commonJsonRawJson.errorSummaryTitle;
```

Comment:
A lot of constants here that I'm also seeing in other files. Please try to define constants once and import where needed